src/rust_codegen.rs

```diff
@@ -519,6 +561,7 @@ pub fn generate_rpi_rec(
         addr_type: addr_type.clone(),
         value_type: value_type.clone(),
         prelude: use_statements(),
+        reg_size_threshold: 64,
```

Comment:
Let's plumb this all the way out, so the driver for the RSF code generator can decide.